ogr/ogrspatialreference.cpp

```diff
@@ -11214,6 +11216,22 @@ OGRErr OGRSpatialReference::importFromEPSGA(int nCode)
                     proj_list_get(d->getPROJContext(), list, 0);
                 if (nonDeprecated)
                 {
+                    if (pszUseNonDeprecated == nullptr)
+                    {
+                        const char *pszNewAuth =
+                            proj_get_id_auth_name(nonDeprecated, 0);
+                        const char *pszNewCode =
+                            proj_get_id_code(nonDeprecated, 0);
+                        CPLError(CE_Warning, CPLE_AppDefined,
+                                 "CRS EPSG:%d is deprecated. "
+                                 "Its non-deprecated replacement %s:%s "
+                                 "will be used instead.\n"
```

Comment:
@rouault what would you think about removing the newline here and rely on applications to format the message?

Reply:
done